apps/rush-lib/src/api/RushConfigurationProject.ts

```diff
@@ -377,15 +411,28 @@ export class RushConfigurationProject {
           case DependencySpecifierType.Version:
           case DependencySpecifierType.Range:
             if (semver.satisfies(localProject.packageJson.version, dependencySpecifier.versionSpecifier)) {
-              localDependencyProjects.push(localProject);
+              yield localProject;
             }
             break;
           case DependencySpecifierType.Workspace:
-            localDependencyProjects.push(localProject);
+            yield localProject;
             break;
         }
       }
     }
-    return localDependencyProjects;
+  }
+
+  /**
+   * Compute the local rush projects that immediately depend on this project
+   */
+  private *_getLocalDependentProjects(): Iterable<RushConfigurationProject> {
+    for (const projectName of this.downstreamDependencyProjectSet) {
+      const localProject: RushConfigurationProject | undefined = this._rushConfiguration.getProjectByName(
+        projectName
+      );
+      if (localProject && localProject.localDependencyProjectSet.has(this)) {
+        yield localProject;
+      }
+    }
```

Comment:
Just return an array.

Reply:
Switched to returning a `Set<RushConfigurationProject>` since the caller directly does so anyway.